django/contrib/gis/db/models/fields.py

```diff
@@ -444,4 +443,5 @@ def get_transform(self, name):
             )
         except ValueError:
             pass
-        return super().get_transform(name)
+        else:
```

Comment:
Is some test failing without this change? I don't see how it's related to this PR.